client/src/app/shared/execution-card/execution-card.component.html

```diff
@@ -17,6 +15,7 @@
     </ng-container>
     {{ execution.name || '#' + execution.id }}
   </div>
+  <mat-icon *ngIf="hasOutput$ | async" style="float:right">playlist_add_check</mat-icon>
```

Comment:
This LGTM, we just need to come up with a different style here :)

Maybe we can use some flexbox-fu to align the icon accordingly.

Reply:
Do you like the icon btw? Or do you prefer a other one?